python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -173,6 +173,10 @@ def handle_pip_compile_errors(error)
             raise GitDependenciesNotReachable, url
           end
 
+          raise Dependabot::OutOfDisk if error.message.end_with?("[Errno 28] No space left on device")
```

Comment:
This is fine or we can also just check for error message contains `No space left on device`.

Reply:
should the message be a constant?

Also purely curious; do we know what is filling up the disk space? I.E., why are the machines running out of disk space?

Reply:
> should the message be a constant?

IMO, not really since that adds an extra layer of indirection when reading the code. Things that would nudge me there are:
- We used this specific message in more than one place with the python handling code
- The match was a complicated regex or had a non-obvious connection to the error so would benefit from a more descriptive constant name.

> Also purely curious; do we know what is filling up the disk space? I.E., why are the machines running out of disk space?

Primary usage of diskspace:
- The repo being updated, especially in ecosystems where we need to clone the repo
- Dependencies, especially when the dependencies are vendored so we'll have old + new versions of the dependencies on disk somewhere.

There's also potential for inefficiency when we update multiple dependencies. If the package manager is leaving behind files for each update that may also contribute to running out of space towards the end of a job. I think we'll be able to better analyze once we start reporting the error.

Reply:
Thanks; nbd with the constant if this is a localized single value; I raised it because I saw a couple scenarios wherein the strings were used as constants.

I'll be excited to see what the data shows up once the reports start coming in! Thanks for fixing this again.